capa/features/extractors/base_extractor.py

```diff
@@ -303,6 +307,13 @@ class DynamicExtractor(FeatureExtractor):
     This class is not instantiated directly; it is the base class for other implementations.
     """
 
+    @abc.abstractmethod
+    def get_base_address(self) -> Union[AbsoluteVirtualAddress, capa.features.address._NoAddress]:
+        """
+        fetch the preferred load address at which the sample was analyzed.
+        """
+        raise NotImplementedError()
+
```

Comment:
i suspect that we may not need this information. do you have a plan for it? if we don't need it now, lets remove this from the interface until we do.

Reply:
its used in the static feature extractor to render output, since there may be relative virtual addresses that need to be resolved. in the dynamic extractor, i can't immediately think of where we'd use this info. can you?